traffic_portal/app/src/common/modules/form/cacheGroup/form.cacheGroup.tpl.html

```diff
@@ -128,10 +128,29 @@
                     <input type="checkbox" name="GeoEnabled" ng-model="localizationMethods.GEO"> Geo-IP Database</input>
                 </div>
             </div>
+            <div class="form-group" ng-class="{'has-error': hasError(cacheGroupForm.fallbacks), 'has-feedback': hasError(cacheGroupForm.fallbacks)}" ng-hide="!isEdgeLoc(cacheGroup.typeId)">
+                <label class="control-label col-md-2 col-sm-2 col-xs-12">Failover Locations</label>
+                <div class="col-md-10 col-sm-10 col-xs-12">
+                    <select name="fallbackOptions" class="form-control" ng-model="fallbackSelected" ng-options="cachegroup.group as cachegroup.group for cachegroup in cacheGroupFallbackOptions" ng-change="updateFallbacks(cacheGroup)">
+                        <option value="">Add Failover Cache Groups...</option>
+                    </select>
+                    <div class="allFallbacks" id="allFallbacks">
+                        <div class="fallback" ng-repeat="fb in cacheGroup.fallbacks" dnd-enable="true" pageid="cacheGroupFallback">
+                            <i class="fa fa-ellipsis-v fallback-grip drop-child" aria-hidden="true"></i>
+                            <div class="fallback-list drop-child">{{ $index + 1 }}.     {{fb}}</div>
```

Comment:
this should probalby be like:

`{{ $index + 1 }}&nbsp;&nbsp;&nbsp;&nbsp;{{fb}}`

Reply:
done